packages/common/container-definitions/src/loader.ts

```diff
@@ -300,6 +300,14 @@ export type ConnectionState =
 	| ConnectionState.CatchingUp
 	| ConnectionState.Connected;
 
+/**
+ * @deprecated Will be removed in future major release. Migrate all usage of IFluidRouter to the "entryPoint" pattern. Refer to Removing-IFluidRouter.md
```

Comment:
```suggestion
 * @deprecated Will be removed in future major release. Migrate all usage of "request" to the "entryPoint" pattern. Refer to Removing-IFluidRouter.md
```

Reply:
Also, add a comment explaining the scenario that this should be used and when it can be removed.